src/components/Insights/TypeaheadSearch.js

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 import Autosuggest from 'react-autosuggest';
 import { fromMapToArray } from './shared';
-import match from 'autosuggest-highlight/match';
-import parse from 'autosuggest-highlight/parse';
+import request from 'request';
 import '../../styles/Insights/TypeaheadSearch.css';
 
+const SUGGESTION_TO_ICON = {
+  'Location': 'fa fa-map-marker fa-2x',
+  'Term': 'fa fa-tag fa-2x',
+};
+
+function fetchLocationsFromFeatureService(bbox, matchName, callback) {
```

Comment:
can you move this function under the `services` folder

Reply:
This is a fairly specific function so there won't be much potential for re-using it, but sure. 9fece04